src/explore-education-statistics-common/src/components/AccordionSection.tsx

```diff
@@ -53,6 +55,7 @@ const AccordionSection = ({
   heading,
   headingTag = 'h2',
   id = 'accordionSection',
+  anchorLinkIdPrefix = 'section',
```

Comment:
Didn't say we should change the changes for `anchorLinkUrl`. Those should still be the same as they were previously i.e. a callback function